tests/help/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Check commands help
+environment:
+  CMD/abort: abort
+  CMD/changes: changes
+  CMD/find: find
+  CMD/install: install
+  CMD/interfaces: interfaces
+  CMD/remove: remove
+execute: |
+  echo "Checking help for command $CMD"
+  expected="(?s)Usage:\n  snap \[OPTIONS\] $CMD.*?\n\nThe $CMD command .*?\nHelp Options:\n  -h, --help +Show this help message\n.*?"
+  actual=$(snap $CMD --help)
+  echo "$actual" | grep -Pzq "$expected" || exit 1
```

Comment:
@zyga The expect tool is a nice way to drive an interactive application, but the snap tool is in the vast majority of cases non-interactive, and this example here we're pretty much simply applying a regular expression to the output. So I'd see expect as one more thing to learn about and to go wrong, instead of an improvement.

Happy to be proven wrong with an actual example, though.